egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -116,6 +116,22 @@ def GetArgs():
     parser.add_argument("--use-presoftmax-prior-scale", type=str, action=nnet3_train_lib.StrToBoolAction,
                         help="if true, a presoftmax-prior-scale is added",
                         choices=['true', 'false'], default = True)
+    # arguments for ephemeral connection
+    parser.add_argument("--add-ephemeral-connection", type=str, action=nnet3_train_lib.StrToBoolAction,
+                        help="if true, a ephemeral connection with dropout connection is added to all layers.",
+                        choices=['true', 'false'], default = False)
+    parser.add_argument("--num-skips-for-ephemeral", type=int,
```

Comment:
--num-layer-skips-for-ephemeral may be better, in BLSTM we may intro ephemeral in time axis, we should prevent ambiguity